frontend/website/pages/docs/tokens.mdx

```diff
@@ -0,0 +1,102 @@
+
+## Creating Tokens
+
+With the CLI, you are able to create your own tokens on Coda. Non-default tokens are a new type of token that is introduced to the Coda blockchain. These new types of tokens have a unique identifier that differentiate them from all other tokens in the blockchain. Users with an associated token account can then receive/send this new type of token that is totally separate from the native Coda token itself!  When you create a new token, you are then able to mint more of the token to be used (see below in [Minting Tokens](/docs/tokens#minting-tokens)).
+
+When creating a new token, a new token ID is created as well as a token account for your public key. This token account owns the token and can mint more of the token. The account will also (eventually) have the power to enable/disable accounts, and to set whether new accounts may be created for the token.
+
+Please note that creating new tokens requires an additional fee on top of the ever-present transaction fee. This additional fee is the fee that must be paid when creating a new token account. This means there is a fee for the account creation on top of the transaction fee that is normally applied.
+
+### Create a new token
+
+```
+$ coda client create-token -sender <PUBLIC_KEY>
+```
+
+<Alert>
+
+The required field for creating a new token account is: 
+
+- -**sender** : The public key from which you want to send the transaction
+
+</Alert>
+
+See 
+```
+$ coda client create-token -help
+```
+to learn more about how to use the command.
+
+## Creating Token Accounts
+
+With the CLI, you are able to create a token account for existing tokens. Token accounts are like regular accounts but solely meant to interact with the non-default tokens that exist in the Coda protocol. You must create a separate unique token account for each type of token you wish to interact with.
+<Alert kind="warning">
+
+To send or receive an existing token, you must have a token account with the specified token. If there is no valid token account with the token, the transaction will fail.
+
+</Alert> 
+
+### Create a new token account 
+
+```
+$ coda client create-token-account -token-owner <PUBLICKEY> -receiver <PUBLICKEY> -sender <KEY> -token <TOKEN_ID>
+```
+
+<Alert>
+
+ If the token owner is not given, the CLI will query to find the token owner. If no owner is found, the command will fail with an error.
+
+The required fields for creating a new token account are:
+
+- -**receiver** : The public key to create the new account for, you can reuse your normal public key here
+
+- -**sender** : The public key for which you want to create the account
```

Comment:
This is confusing as sender seems to be the same as the receiver. As per the help isn't this `Public key from which you want to send the transaction` and is equivalent to `feePayer` in GraphQL